react/features/base/media/reducer.js

```diff
@@ -57,6 +59,43 @@ function _audio(state = AUDIO_INITIAL_MEDIA_STATE, action) {
     }
 }
 
+/**
+ * Initial state of the sounds reducer which is a {@code Map} of globally stored
+ * sounds.
+ *
+ * @type {Map<string, AudioElement>}
+ */
+const SOUNDS_INITIAL_STATE = new Map();
```

Comment:
You don't need to have any initial state, undefined will work just fine with how you're handling the actions.